modules/apps/saved-content/saved-content-taglib/src/main/java/com/liferay/saved/content/taglib/servlet/taglib/SavedContentTag.java

```diff
@@ -85,8 +90,34 @@ public void setCompanyId(long companyId) {
 		_companyId = companyId;
 	}
 
-	public void setContentTitle(String contentTitle) {
-		_contentTitle = contentTitle;
+	public void setContentTitle(Locale locale) {
```

Comment:
Now, this method doesn't follow a standard `set` method where the new value is passed as a parameter.